src/plugins/auto/auto_schedule.cpp

```diff
@@ -175,15 +174,18 @@ void AutoSchedule::init(const ScheduleContext::Ptr& sContext) {
     }
     LOG_INFO_TAG("select device:%s", _loadContext[ACTUALDEVICE].deviceInfo.deviceName.c_str());
     bool isActualDevCPU =
-        _loadContext[ACTUALDEVICE].deviceInfo.deviceName.find("CPU") !=std::string::npos;
+        _loadContext[ACTUALDEVICE].deviceInfo.deviceName.find("CPU") !=std::string::npos && !isCumulative;
     // if Actual device is CPU, disabled _loadContext[CPU], only use _loadContext[ACTUALDEVICE]
     if (isActualDevCPU || isCumulative) {
         _loadContext[CPU].isEnabled = false;
+        _autoSContext->_exeDevices = isActualDevCPU ? "CPU"
+                        : _loadContext[ACTUALDEVICE].deviceInfo.deviceName.substr(_loadContext[ACTUALDEVICE].deviceInfo.deviceName.find(":") + 1);
```

Comment:
In cumulative throughput mode, the executing device should remove the device that failed to load the network.

Reply:
ctput will load to multi:*,* for multi, there is no logic to check load failure

Reply:
Will follow up in next PR